Swift/Helpers/FSExtensions/FSE+String.swift

```diff
@@ -71,6 +71,29 @@ public extension String {
         return ceil(rect.size.height)
     }
     
+    public func fs_stringBetweenString (firstString: String, secondString: String) -> String? {
+        var string: String?
```

Comment:
You must add initial value because  in 89 row you use force unwrapping
This code must work but other programmer can not see this and leave nil value in string variable